src/app/shared/components/autocomplete/autocomplete.component.scss

```diff
@@ -0,0 +1,3 @@
+.autocomplete-input::-webkit-calendar-picker-indicator {
```

Comment:
let's move that into _global.scss and remove this file and import please 
```(styleUrls: ['./autocomplete.component.scss'],)```
It is best to rely on plain sass/css stylesheet than creating a dependency with a framework and inlined styling that is difficult to port over.

Reply:
@JonLazariniDD  i suggested this approach for @rspasici , as an update to do. It will make the autocomplete complete self contained and with its own single responsibility. The extra css added should not be global. I think it should be left in place in `styleUrls: ['./autocomplete.component.scss']`

Reply:
When I said in the global scss i meant loaded as a separate entity (It could be a partial) for example or in there as the file is quite small and managed to use the css lib really well plus this autocomplete component is a reusable component. 
I am against having inlined/scoped css especially when we use a css lib (which is plain css). If for some some reason it was decided to switch from Angular to React (or any other js framework in the future), this style would be lost as it is part/coupled to a component now whereas if it was contained within a compiled stylesheet, we could simply load the css file (containing all the styling) in the new framework and use the same html markup but wrapped within whatever new Js framework existing.
I think the css is pretty tidy now, the govuk lib pretty clear and I don't see the real benefit of scoping css into components.

Reply:
Why did we resolve that?